corehq/apps/settings/forms.py

```diff
@@ -72,14 +73,13 @@ def save(self, commit=True):
 
 
 class HQPhoneNumberMethodForm(PhoneNumberMethodForm):
-    number = forms.CharField(label=_("Phone Number"),
-                             validators=[validate_international_phonenumber],
+    number = forms.CharField(required=False,
+                             label=_('Phone Number'),
                              widget=forms.TextInput(
                                  attrs={'placeholder': _('Start with +, followed by Country Code.')}))
+    number.run_validators = validate_international_phonenumber
```

Comment:
Any reason to not use the ```validators``` kwarg?
https://docs.djangoproject.com/en/1.11/ref/validators/#module-django.core.validators

Reply:
Yes: the validators are all run through this generic logic in 'Field':

https://github.com/django/django/blob/master/django/forms/fields.py#L129

But I wasn't able to use that because:
a) I need to allow empty responses for the back button to work (because that's technically 'submitting' this form using form wizard logic)
b) I need the error message to pop up when there is an empty response and 'next' is clicked, which wouldn't happen because empty fields are considered valid responses in the above logic

So instead of using a custom validator I have to overwrite the validation script